1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -5,19 +5,26 @@
 */
 
 const BIRTHDAYS = [
-    "January 7th",
-    "February 12th",
-    "April 3rd",
-    "April 5th",
-    "May 3rd",
-    "July 11th",
-    "July 17th",
-    "September 28th",
-    "November 15th"
+  "January 7th",
+  "February 12th",
+  "April 3rd",
+  "April 5th",
+  "May 3rd",
+  "July 11th",
+  "July 17th",
+  "September 28th",
+  "November 15th",
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  // TODO
+  let i = 0;
+   while (i < birthdays.length) {
+    if (birthdays[i][1] === "u") {
```

Comment:
This works, but what if later on more birthdays are added to the array including ones in June? Is it a bit more robust and readable to look for the whole word `July`?